resources/monitoring/charts/alert-rules/README.md

```diff
@@ -2,12 +2,33 @@
 
 ## Overview
 
-Kyma uses Prometheus alert rules for monitoring the health of its resources. This chart is the starting point for configuring alert rules.
+Kyma uses Prometheus alert rules to monitor the health of resources. Use this chart to configure alert rules.
 
+## Details
 
-### Creating Alert Rules in Kyma
+### Alert rules
 
-Prometheus uses the a label selector **spec.ruleSelector** to identify those ConfigMap that holding Prometheus rule files.
+You can define the following alert rules:
+
+- Alert when a Pod is not running
+
+    The Alertmanager sends out alerts when one of the Pods is not running in `kyma-system`, `kyma-integration`, `istio-system`, `kube-public`, or `kube-system` Namespace.
+
+- Monitor Persistent Volume Claims (PVC)
+
+    The Alertmanager sends out alerts when the PVC exceeds 90% for the following system Namespaces: `kyma-system`, `kyma-integration`, `heptio-ark`, `istio-system`, `kube-public`, or `kube-system`. To avoid this, increase the capacity of PVC.

```

Comment:
```suggestion
    The Alertmanager sends out alerts when the PVC exceeds 90% for the following system Namespaces: `kyma-system`, `kyma-integration`, `heptio-ark`, `istio-system`, `kube-public`, or `kube-system`. To avoid this, increase the capacity of the PVC.
```

Reply:
Done.